src/stores/tutorial-store.js

```diff
@@ -0,0 +1,87 @@
+import { observable, action, computed } from 'mobx';
+
+export default class TutorialStore {
+    @observable showAgain = true;
+    @observable modalIsOpen = true;
+    @observable strategy = 'martingale';
+    @observable tradeOptions = {
+        market  : 'm1',
+        contract: 'c1',
+        stake   : 0,
+        size    : 0,
+        loss    : 0,
+        profit  : 0,
+    };
+
+    constructor() {
+        this.showAgain = localStorage.getItem('StarterHint') === 'true' || true;
+        this.modalIsOpen = this.showAgain;
+    }
+
+    changeStrategy = async () => {
+        const workspace = Blockly.mainWorkspace;
+        const strategy_xml = await fetch(`dist/${this.strategy}.xml`).then(response => response.text());
+        const strategy_dom = Blockly.Xml.textToDom(strategy_xml);
+
+        const modifiedValueBlock = type => {
+            const valueBlock = strategy_dom.querySelectorAll(`value[id="${type}_value"]`)[0];
+            valueBlock.innerHTML = `<block type="math_number"><field name="NUM">${this.tradeOptions[type]}</field></block>`;
+        };
+
+        const modifiedValue = (name, type) => {
+            const block = strategy_dom.querySelectorAll(`field[name="${name}"]`);
```

Comment:
Same here. 

- Please use underscores for variable definitions, and add a new line after.
- If we just want to select the first matching element, we can use `querySelector` instead of `querySelectorAll`.